shared/git/index.tsx

```diff
@@ -2,86 +2,97 @@ import * as React from 'react'
 import * as Kb from '../common-adapters'
 import * as Styles from '../styles'
 import Row from './row/container'
+import {union} from '../util/set'
 
 export type Props = {
   error?: Error
-  expandedSet: Set<string>
   loading: boolean
+  initialExpandedSet?: Set<string>
   onShowDelete: (id: string) => void
   onNewPersonalRepo: () => void
   onNewTeamRepo: () => void
-  onToggleExpand: (id: string) => void
   personals: Array<string>
   teams: Array<string>
 }
 
-class Git extends React.Component<Props & Kb.OverlayParentProps, {}> {
-  private menuItems: Kb.MenuItems = [
-    {
-      icon: 'iconfont-person',
-      onClick: () => this.props.onNewPersonalRepo(),
-      title: 'New personal repository',
-    },
-    {
-      icon: 'iconfont-people',
-      onClick: () => this.props.onNewTeamRepo(),
-      title: 'New team repository',
-    },
-  ]
-
-  rowPropsToProps = (id: string) => ({
-    expanded: this.props.expandedSet.has(id),
-    id,
-    onShowDelete: this.props.onShowDelete,
-    onToggleExpand: this.props.onToggleExpand,
-  })
+// keep track in the module
+let moduleExpandedSet = new Set<string>()
```

Comment:
why not in redux store?

Reply:
1. this was moved from elsewhere
2. i think state that no other component cares about and has no benefit from being in the store likely shouldn't be in the store. It's purely local state. We just happen to want it to live past unmounting